src/AppBundle/Controller/DefaultController.php

```diff
@@ -16,13 +15,8 @@ class DefaultController extends Controller
     public function userDeliveries($user_id)
     {
         if ($user_id){
-            $user = $this->getDoctrine()->getRepository(User::class)->find($user_id);
-
-            if(!$user)
-                throw new NotFoundHttpException('user not found');
-
             $customerDeliveriesQueryHandler = new CustomerDeliveriesQueryHandler();
-            $deliveries = $customerDeliveriesQueryHandler->onCustomerDeliveriesQuery($user);
+            $deliveries = $customerDeliveriesQueryHandler->onCustomerDeliveriesQuery($user_id);
```

Comment:
You have defined your service in the services.yml meaning that you should not create a new instance of it. Symfony will do it for you. $deliveries = $this->container->get('app.customer_deliveries_query-handler');